### ISSUE TITLE
win_iis_webbinding requires 'ip' property to be set despite it being optional

### ISSUE THREAD

**Issue by nathanwebsterdotme** [ansible/ansible-modules-extras#2614]
##### ISSUE TYPE
- Bug Report
##### COMPONENT NAME

win_iis_webbinding
##### ANSIBLE VERSION

```
ansible 2.1.0.0
```
##### CONFIGURATION

No changes
##### OS / ENVIRONMENT

Running from: OSX El Capitan 10.11.5
Managing: Windows Server 2012 R2
##### SUMMARY

win_iis_webbinding fails run if 'ip' property isn't set.  This worked in Ansible 2.0.0.2.
##### STEPS TO REPRODUCE

Run a play using win_iis_webbinding without the 'ip' property set.

```
 - name: IIS | Add HTTPS bindings as required
      win_iis_webbinding:
        name: "{{ item.name }}"
        host_header: "{{ item.hostname }}"
        protocol: https
        port: 443
        state: present
      with_items: "{{ websites }}"
      when: item.ssl == true
```
##### EXPECTED RESULTS

win_ip_webbinding should still set the host header even if 'ip' isn't set.  There is a condition in the web_ip_webbinding.ps1 that checks for an empty value and defaults it to 0.0.0.0 but this condition is failing. 
##### ACTUAL RESULTS

```
An exception occurred during task execution. The full traceback is:
At C:\Users\Administrator\AppData\Local\Temp\ansible-tmp-1469183297.15-215721360802217\win_iis_webbinding.ps1:281 char:5
+ If ($params.ip) {
+     ~~~~~~~~~~
failed: [*****] (item={u'name': u'*****', u'hostname': u'*****', u'ssl': True, u'state': u'started', u'version': u'*****', u'managedRuntimeVersion': u'v2.0'}) => {"failed": true, "invocation": {"module_name": "win_iis_webbinding"}, "item": {"hostname": "*****", "managedRuntimeVersion": "v2.0", "name": "*****", "ssl": true, "state": "started", "version": "*****"}, "msg": "The property 'ip' cannot be found on this object. Verify that the property exists."}
```

**Comment by nathanwebsterdotme** [comment]
Workaround is to set the following property:
`
ip: '*'
`

**Comment by jhawkesworth** [comment]
Hello,

Thanks for this.  I'm pleased to say this is fixed in latest devel and 2.1.1 rc3.

If you would like to try out the release candidate there are instructions for installing it here: https://groups.google.com/forum/#!topic/ansible-devel/sRwwjub5HDQ

Since a fix has been made, and will be in the next release, I'm going to close this issue.
